src/blackmisc/simulation/fscommon/fsdirectories.cpp

```diff
@@ -88,6 +89,59 @@ namespace BlackMisc::Simulation::FsCommon
         return dir;
     }
 
+    QString msfsDirImpl()
+    {
+        QStringList someDefaultDirs(
+            {
+                // C:\Users\rolan\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\
+            // todo msfs: add default simulator directory
+            });
+        const QStringList locations = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation);
+        for (const QString &path : locations)
+        {
+            const QString msfsPackage = CFileUtils::appendFilePaths(CFileUtils::appendFilePaths(path, "Packages"), "Microsoft.FlightSimulator_8wekyb3d8bbwe");
+            const QDir d(msfsPackage);
+            if (!d.exists()) { continue; }
+            return msfsPackage;
+        }
```

Comment:
This likely only works for some installation methods of MSFS (according to https://helpdesk.aerosoft.com/hc/en-gb/articles/5023507568925-How-to-locate-the-Community-folder-in-Microsoft-Flight-Simulator only for Microsoft Store installations).

Other installation directories might be harder to locate automatically (perhaps through some registry keys). If this method is merge like this into ``main``, we should open an issue to improve it in the future.